Libraries/WebSocket/WebSocketBase.js

```diff
@@ -33,19 +33,19 @@ class WebSocketBase extends EventTarget {
   readyState: number;
   url: ?string;
 
-  constructor(url: string, protocols: ?any) {
+  constructor(url: string, protocols: ?string | ?Array<string>, options: ?{origin?: string}) {
```

Comment:
This is according to spec.  The WS spec has an optional protocols argument that can be a string or array of strings.

https://html.spec.whatwg.org/multipage/comms.html#the-websocket-interface